exercism: add page (#17170)
tldr-pages/tldr: pages/common/exercism.md

# exercism

> Download coding exercises in any language and submit your solutions.
> More information: <https://exercism.org/docs/using/solving-exercises/working-locally>.

> - Configure exercism following instructions for api token:

Reviewer: ```suggestion
- Configure exercism following instructions for the API token:
```

`exercism configure`

> - Download hello-world for any programming language into exercism workspace:

Reviewer: ```suggestion
- Download `hello-world` for any programming language into exercism workspace:
```

> `exercism download --track={{lang}} --exercise=hello-world`

Reviewer 2: ```suggestion
`exercism download --track {{lang}} --exercise hello-world`
```
Does the command work without `=`?

Author: yes it does

> - Open an exercise dir on the website:

Reviewer 2: ```suggestion
- Open an exercise directory on the website:
```

> `exercism open {{path/to/solution/}}`

Reviewer 2: We often use `{{path/to/directory}}` unless the syntax requires it like in the case of `rsync` https://github.com/tldr-pages/tldr/blob/main/pages/common/rsync.md?plain=1#L25

> - Test your solution. It will also tell you if you are missing tools for the {{lang}}:

Reviewer: ```suggestion
- Test your solution. It will also tell you if you are missing tools for the programming language:
```
